src/components/_list-view.scss

```
@@ -731,7 +731,9 @@ $list-view-gaps: $gaps;
 }
 
 .ncgr-list-tile__leading {
-  width: 3.5rem;
+  box-sizing: border-box;
+  min-width: 3rem;
+  padding-right: 1rem;
```

チェックボックス、まだ離れてそうなので、これくらいでもよさそうかも

```suggestion
  min-width: 2.5rem;
  padding-right: 0.5rem;
```